src/BenchmarksApps/MapAction/Program.cs

```diff
@@ -27,21 +27,24 @@
                 endpoints.MapGet("/json", (Func<object>)Json);
 
                 // Parameterized plain-text endpoint
-                string SayHello(string name) => $"Hello, {name}!";
+                string SayHello(string name, int age, string location) => $"Hello, {name}! You're {age} years old and based in {location}.";
                 // With no filters
-                endpoints.MapGet("/hello/{name}", SayHello);
+                endpoints.MapGet("/hello/{name}/{age}/{location}", SayHello);
                 // With a filter that no-ops
-                endpoints.MapGet("/helloEmptyFilter/{name}", SayHello)
+                endpoints.MapGet("/helloEmptyFilter/{name}/{age}/{location}", SayHello)
```

Comment:
Does this work? We need  anew handler no?

Reply:
The `plaintextNoParamsWithFilter` validates a pass through filter without parameters. This one validates a pass through filter with parameters.

Reply:
But the method doesn't have the arguments. Can we add another handler?

Reply:
What do you mean the method doesn't have arguments? Are you referring to `SayHello`?

Reply:
David and I synced about it only. He thought I was reusing the `SayHello` handler for the `plainTextNoParams` scenarios.